sonic_installer/common.py

```diff
@@ -20,10 +20,14 @@
 DOCKERDIR_NAME = 'docker'
 
 # Run bash command and print output to stdout
-def run_command(command):
-    click.echo(click.style("Command: ", fg='cyan') + click.style(command, fg='green'))
-
-    proc = subprocess.Popen(command, shell=True, text=True, stdout=subprocess.PIPE)
+def run_command(command, devnull=False):
```

Comment:
>devnull=False [](http://example.com/codeflow?start=25&length=13)

Use `stdout=subprocess.PIPE` will simplify the code.

Reply:
devnull is for these original commands
`run_command("docker exec -i teamd pkill -USR1 teamd > /dev/null")`
`run_command("docker kill %s > /dev/null" % container_name)`